tests/pdks/test_mos.py

```diff
@@ -39,7 +39,6 @@ def build_test(pdk, prim, *, n, X, Y):
                                 "CMFB_{}",
                                 "SCM_{}",
                                 "CMC_{}",
-                                "CMC_{}_S",
```

Comment:
I think we are missing a test for CMC_S_{} ? Do we also need to test the _B variants?

Reply:
@arvuce22 this may need some action as well